subsys/net/lib/lwm2m/lwm2m_shell.c

```diff
@@ -91,18 +76,20 @@ static int cmd_send(const struct shell *sh, size_t argc, char **argv)
 		return -E2BIG;
 	}
 
-	for (int i = ignore_cnt; i < path_cnt; i++) {
+	for (int i = 0; i < path_cnt; i++) {
 		/* translate path -> path_obj */
-		ret = lwm2m_string_to_path(argv[i], &lwm2m_path_list[i], '/');
+		const char *p = argv[1 + i];
+
+		ret = lwm2m_string_to_path(p, &lwm2m_path_list[i], '/');
 		if (ret < 0) {
 			return ret;
 		}
 	}
 
-	ret = lwm2m_send(ctx, lwm2m_path_list, path_cnt, confirmable);
+	ret = lwm2m_send(ctx, lwm2m_path_list, path_cnt, true);
```

Comment:
Shouldn't this use `lwm2m_send_cb()` now?

Reply:
Done